clients/python/Makefile

```diff
@@ -18,8 +18,12 @@ deploy-latest-mr:
 	cd ../../ && IMG_VERSION=${IMG_VERSION} make image/build && LOCAL=1 ./scripts/deploy_on_kind.sh
 	kubectl port-forward -n kubeflow services/model-registry-service 8080:8080 &
 
+.PHONY: deploy-local-registry
+deploy-local-registry:
+	cd ../../ && ./scripts/deploy_local_kind_registry.sh
+
 .PHONY: test-e2e
-test-e2e: deploy-latest-mr
+test-e2e: deploy-latest-mr deploy-local-registry
```

Comment:
can you kindly doublecheck this is consistent with the approach from https://github.com/kubeflow/model-registry/pull/765/files#diff-e01c397ab995b0ee8633b9be78064e1cff0b691f10166df64614dc0bb2b5395eR27 please?